examples/problem_decomposition/ifci_mifno.ipynb

```diff
@@ -0,0 +1,554 @@
+    "\\end{align*}\n",
+    "$$\n",
+    "\n",
+    "where $\\epsilon_i$, $\\epsilon_{ij}$, $\\epsilon_{ijk}$, and $\\epsilon_{ijkl}$ are, respectively, the one-, two-, three-, and four-body increments (expansions) defined as\n",
+    "\n",
+    "$$\n",
+    "\\begin{align*}\n",
+    "\\epsilon_i &= E_c(i) \\\\\n",
+    "\\epsilon_{ij} &= E_c(ij) - \\epsilon_i - \\epsilon_j \\\\\n",
+    "\\epsilon_{ijk} &= E_c(ijk) - \\epsilon_{ij} - \\epsilon_{ik} - \\epsilon_{jk} - \\epsilon_{i} - \\epsilon_{j} - \\epsilon_{k} \\\\\n",
+    "\\epsilon_{ijkl} &= E_c(ijkl) - \\epsilon_{ijk} - \\epsilon_{ijl} - \\epsilon_{jkl} - \\dots \\\\\n",
+    "&\\vdots\n",
+    "\\end{align*}\n",
+    "$$\n",
+    "\n",
+    "The following figure, taken from [J. Chem. Phys. 2021, 155, 034110](https://doi.org/10.1063/5.0054647), illustrates this problem decomposition scheme in terms of 1-body and many-body interactions. On each subproblem, a truncation is applied to reduce their virtual space. The subproblems resulting from the iFCI and MI-FNO reduction can then be solved by any algorithm, including quantum algorithms such as the phase estimation algorithm and the variational quantum eigensolver (VQE), to predict the correlation energies of a molecular system.\n",
+    "\n",
+    "<div>\n",
+    "<img src=\"../img/mi.png\"/  width=\"700\">\n",
+    "</div>\n",
+    "\n",
+    "The iFCI and MI-FNO problem decomposition pipelines are available in QEMIST Cloud. In this notebook, we illustrate how to export MI fragment data computed in QEMIST Cloud, and import it in Tangelo for further treatment, such as using quantum solvers."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Performing MI calculations with QEMIST Cloud\n",
+    "\n",
+    "[QEMIST Cloud](https://goodchemistry.com/qemist-cloud/) is an engine that enables faster, more accurate, and scalable ways to perform computational chemistry simulations. This platform leverages easily and readily accessible computers on the cloud to perform chemistry simulations that were previously intractable even on expensive, high-performance computing environments.\n",
+    "\n",
+    "In order to leverage this platform to perform the MI calculations, subscribing to the services and installing the `qemist_client` python package is necessary. This notebook does not require either of these things: we provide the code snippet used to generate the problem decomposition results, and use those as pre-computed values for the rest of the notebook.\n",
+    "\n",
+    "For more information about QEMIST Cloud (installation, features, issues, `qemist_client` API...), please refer to the QEMIST Cloud documentation or contact the development team.\n",
+    "\n",
+    "In the script below, each fragment's virtual space is truncated to keep only the virtual orbitals with the highest occupation number. MI is paired with the Heath-Bath Configuration Interaction (HBCI) classical solver."
```

Comment:
You have not mentioned SNO until now (i suggested mentioning it above because FNO vs SNO). I think you should drop a sentence here to say one can do MIFNO if FNO is used, and iFCI if SNO is used, and that the cell below show both versions with iFCI commented